docs/snapshots.asciidoc

```diff
@@ -16,7 +16,7 @@ For more information on Elasticsearch snapshots, see https://www.elastic.co/guid
 [id="{p}-install-plugin"]
 ==== Install the storage repository plugin
 
-To install the storage repository plugin, you can either use a custom image or add your own init container which
+To install the storage repository plugin, you can either use a link:k8s-images.html[custom image] or add your own init container which
```

Comment:
Same here, add a link to the k8s-init-containers-plugin-downloads.html doc page?